docs/docusaurus/dids/create.md

```diff
@@ -14,47 +14,47 @@ Once the create-operation gets constructed, deriving a DID from this operation i
 
-PRISM Agent provides two endpoint groups to facilitate PRISM DID usage.
+The Cloud Agent provides two endpoint groups to facilitate PRISM DID usage.
 
 - `/dids/*`
 facilitate of low-level interactions between DID operations and the blockchain.
 The DID controllers will handle key management independently and use these endpoints for blockchain interaction.
 
 - `/did-registrar/*`
-Facilitates a higher-level interaction with PRISM DID, where the PRISM Agent handles key management concerns.
+Facilitates a higher-level interaction with PRISM DID, where the Cloud Agent handles key management concerns.
 
 ## Endpoints
 
 The example uses the following endpoints
 
-| Endpoint                                                                               | Description                                         | Role           |
-|----------------------------------------------------------------------------------------|-----------------------------------------------------|----------------|
-| [`GET /did-registrar/dids`](/agent-api/#tag/DID-Registrar/operation/listManagedDid)    | List all DIDs stored in PRISM Agent                 | DID Controller |
-| [`POST /did-registrar/dids`](/agent-api/#tag/DID-Registrar/operation/createManagedDid) | Create a new PRISM DID to be managed by PRISM Agent | DID Controller |
-| [`GET /dids/{didRef}`](/agent-api/#tag/DID/operation/getDid)                           | Resolve a DID to DID document representation        | DID Controller |
+| Endpoint                                                                               | Description                                             | Role           |
+|----------------------------------------------------------------------------------------|---------------------------------------------------------|----------------|
+| [`GET /did-registrar/dids`](/agent-api/#tag/DID-Registrar/operation/listManagedDid)    | List all DIDs stored in the Cloud Agent                 | DID Controller |
+| [`POST /did-registrar/dids`](/agent-api/#tag/DID-Registrar/operation/createManagedDid) | Create a new PRISM DID to be managed by the Cloud Agent | DID Controller |
+| [`GET /dids/{didRef}`](/agent-api/#tag/DID/operation/getDid)                           | Resolve a DID to DID document representation            | DID Controller |
 
 ## DID Controller interactions
 
-### 1. Check existing DID on the PRISM Agent
+### 1. Check existing DID on the Cloud Agent
 
 ```bash
 curl --location --request GET 'http://localhost:8080/prism-agent/did-registrar/dids' \
   --header "apikey: $API_KEY" \
   --header 'Accept: application/json'
 ```
-The result should show an empty list, as no DIDs exist on this PRISM Agent instance.
+The result should show an empty list, as no DIDs exist on this Cloud Agent instance.
 
-### 2. Create a PRISM Agent managed DID using DID registrar endpoint
+### 2. Create the Cloud Agent managed DID using DID registrar endpoint
```

Comment:
The 'a PRISM Agent' was changed to "the Cloud Agent": please use the original 'a'.